src/lib.rs

```diff
@@ -1782,8 +1915,8 @@ extern "C" fn test_stream_callback(
 
 #[test]
 fn test_module() {
-    let api = Api::new();
-    let registration = Registration::new(&api, ptr::null());
+    let api = Api::new().unwrap();
```

Comment:
If this returns an `Err` will `unwrap` trigger the panic now?

Reply:
I've refactored error handling in tests in the latest commit.